### ISSUE TITLE
Test uploading Required Documents from mobile device

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#4841]
Some customers are having difficulty uploading documents on mobile. 
TODO: test the mobile experience and possibly add language that encourages folks to just take a photo and upload it.

**Comment by dianamartin** [comment]
Testing Set-up
Mobile Device - iOS 12 mini
Browser: Chrome

When going through the application online there definitely is a different experience. We have not optimized the form to be done on a phone or other mobile device. 

This page - `Add Vehicles`
I saw an obvious gap between fields, I think it's bc we hide the other license plates and it's showing here
![image](https://user-images.githubusercontent.com/21112975/106822582-035af000-6645-11eb-9376-8acbe4daf23c.png)

The page - `Add Documents`
![image](https://user-images.githubusercontent.com/21112975/106822633-1241a280-6645-11eb-8d49-9fb57fa621fb.png)

Shows that the fields aren't optimized for mobile users and it's not obvious for a user to click on "Choose File" and then they'll be prompted to "choose a photo" or "take a photo". So perhaps context can be changed here for a better user experience.

**Comment by dianamartin** [comment]
Will test with other browsers tomorrow

**Comment by dianamartin** [comment]
Mobile Device: `iOS 12 mini`
Browser: `Firefox`

**Comment by dianamartin** [comment]
# Testing on Safari - Mobile 
Mobile Device: `iOS 12 mini`
Browser: `Safari`

## RPP Home page
![image](https://user-images.githubusercontent.com/21112975/107236327-37d6ff00-69eb-11eb-9a81-519f8212ee1c.png)

## Get Started page
- We have modal pop-ups enabled for desktop users, but in mobile it shrinks the page real estate quite a bit
- Also easier for a user to click off the page on mobile
![image](https://user-images.githubusercontent.com/21112975/107236383-49200b80-69eb-11eb-8acd-eaa919d934ef.png)

**Comment by dianamartin** [comment]
## Overall Comments
I think we need to re-evaluate the mobile application process and perhaps make optimization decisions so that users have a better experience. I think it would be nice to know how many people their applications from their computers v. mobile device.

**Comment by ChrispinP** [comment]
I'm a bit surprised that Knack is not as optimized as expected. We should be doing what we can to optimize for Mobile. Since you have iOS I did an Android test as well.

**Test Setup:** Android Chrome

1) When entering Name, Address, or License Plates, I experienced a weird interaction where entering 3 or more characters caused the field to randomly copy my previous characters into the field repeatedly. For example If I wanted to enter Chris, I would get CHRCHR. No issue with any other fields like Email or entering # of Day Passes
2) The Spanish Application Form is shoved to the bottom of the page because it is in the second column
![image](https://user-images.githubusercontent.com/67915167/107265512-f99d0800-6a09-11eb-8d82-2658366a43f6.png)
3) The first page of the application, Knack shrinks the column since their are two columns, with the Spanish Application Form in that second column. 
![image](https://user-images.githubusercontent.com/67915167/107266049-9b245980-6a0a-11eb-892e-81d7a8841543.png)
4) Additionally, because we use two columns for license plates as well, Knack shoves the second column underneath the first inclusive of the hidden blank field used for formatting. 
![image](https://user-images.githubusercontent.com/67915167/107266500-3e756e80-6a0b-11eb-9f64-c4d90b236f79.png)
5) We should shrink the amount of space used on the "invoice" page so its printable to a single page. 
![image](https://user-images.githubusercontent.com/67915167/107266744-901df900-6a0b-11eb-8ec9-64eda11eb54d.png)
6) I had no issues with the Required Documents section. I was able to choose an app for an image or access my phones file storage to attach a file. So file issue may be unique to iOS.
7) I was able to use my phones back button to access previous pages, not sure if its possible to disable this.
8) No issue with modals, they take up most of the screen space so i dont close them unless i purposely do so. That said, they look pretty ugly if not using a single column.
8) Should the RPP application be appearing at the top of search results? Most are probably navigating to the application via the City of Austin site. (which i did)

**In Summary**, it seems primarily everything should be single column for mobile but by doing so, this would ruin the way it looks on desktop. So does this mean Desktop and Mobile need different applications and we redirect users based on environment they are on or should we 'listen' for the device being used and adjust the JS/CSS based on the environment?

**Comment by dianamartin** [comment]
@moorerst I think the next steps is to maybe coordinate with the Dev team and see which functionality is the most valuable for mobile device users so we can enhance their experience.

**Comment by moorerst** [comment]
copy styling from signs and markings to test

**Comment by dianamartin** [comment]
Closing this issue, to create a new one for the Mobile Optimization
